js/freewheel.js

```diff
@@ -448,12 +460,11 @@ OO.Ads.manager(function(_, $) {
         }
         else {
           // Trigger the ad
-          _registerDisplayForNonlinearAd();
```

Comment:
These are registered for their corresponding types below. If ad is linear, we register the display for a linear ad on line 466. If the ad is non linear, we register the display on line 495.